server/tests/Routes/meal.routes.test.js

```diff
@@ -92,16 +98,18 @@ describe('/DELETE meal', () => {
       .delete('/api/v1/meals/1114')
       .end((err, res) => {
         expect(res.statusCode).to.equal(404);
-        // expect(res).to.have.status(400);
-        // if (err) return done(err);
+        expect(res.body.message).to.equal(`Cannot find meal with id ${1114}`);
```

Comment:
Is 1114 a variable?

Reply:
No it is not, it is the id parameter I used for my test.